app/src/main/java/com/sunasterisk/fooddaily/data/repository/RecipeRemoteRepository.kt

```diff
@@ -5,16 +5,16 @@ import com.sunasterisk.fooddaily.data.source.OnLoadedCallback
 import com.sunasterisk.fooddaily.data.source.RecipeDataSource
 import com.sunasterisk.fooddaily.data.source.remote.RecipeRemoteDataSource
 
-class RecipeRepository private constructor(
+class RecipeRemoteRepository private constructor(
```

Comment:
```suggestion
class RecipeRepository private constructor(
```
cả local lẫn remote dùng chung 1 cái